src/sign.rs

```diff
@@ -72,3 +72,31 @@ where
         goblin::pe::certificate_table::AttributeCertificateType::PkcsSignedData,
     ))
 }
+/// Produces a certificate for the given PE
+/// with the given signer.
+/// The signer identity is derived from the leaf certificate.
+pub fn create_certificate<'pe, D: Digest, S, Signature>(
+    pe: &PE<'pe>,
+    certificates: Vec<Certificate>,
+    leaf_certificate: Certificate,
+    signer: &S,
+) -> Result<AttributeCertificate<'pe>, SignatureError>
+where
+    D: const_oid::AssociatedOid,
+    S: Keypair + DynSignatureAlgorithmIdentifier,
+    S::VerifyingKey: EncodePublicKey,
+    S: Signer<Signature>,
+    Signature: SignatureBitStringEncoding,
+{
+    create_certificate_with_sid::<D, S, Signature>(
+        pe,
+        certificates,
+        cms::signed_data::SignerIdentifier::IssuerAndSerialNumber(
+            cms::cert::IssuerAndSerialNumber {
```

Comment:
note to self: This could be worth a `impl From<&Cert> for cms::cert::IssuerAndSerialNumber`